INSTALL.md

```diff
@@ -19,8 +19,8 @@ This document provides a guide for intalling AWX.
   - [Start the build](#start-the-build)
   - [Post build](#post-build)
   - [Accessing AWX](#accessing-awx)
-- [Docker](#Docker)
-  - [Prerequisites](#prerequisites-1)
+- [Docker](#docker)
+  - [Prerequisites](#prerequisites-2)
```

Comment:
Based on patterns used elsewhere in document, looks like this should still be `#prerequisites-1`

Reply:
Yeah, #prerequisites-1 takes you to the Prerequisites heading under the OpenShift section, not the one in the Docker section. The other links I think you're thinking of, the ones with -1, are unique to the Docker section of the document.